meteor-app/imports/ui/service/clientOP2.js

```diff
@@ -0,0 +1,164 @@
+/* eslint-disable no-lone-blocks */
+import React from 'react';
```

Comment:
Please rename this file to conform to naming standards

Reply:
this one is done by Carl I think